src/desktop/apps/auction/components/layout/ConfirmRegistrationModal.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useEffect, useState } from "react"
+import { Modal } from "reaction/Components/Modal/Modal"
+import { Serif, Button, CheckCircleIcon, Theme, Box } from "@artsy/palette"
+import { connect } from "react-redux"
+
+const _ConfirmRegistrationModal = ({ me }) => {
+  useEffect(() => {
+    const replaceModalTriggerPath = location.pathname.replace(
+      "/confirm-registration",
+      ""
+    )
+    history.replaceState({}, document.title, replaceModalTriggerPath)
+  }, [])
+  if (!(me && me.bidders && me.bidders.length)) return null
+
+  const bidder = me.bidders[0]
+
+  const [showModal, setShowModal] = useState(true)
+  const hideModal = () => {
+    setShowModal(false)
+  }
+
+  const Content = bidder.qualified_for_bidding
+    ? RegistrationComplete
+    : RegistrationPending
+
+  return (
+    <Theme>
```

Comment:
Is this component rendered outside of the `auction/components/App.js` context?

Reply:
Should have been removed, one push got rejected because I forgot to remove the `Theme` import above. All better.